codegen/generator/nodejs/templates/src/object_comment_test.go

```diff
@@ -25,8 +24,7 @@ func TestComment(t *testing.T) {
 	})
 	t.Run("multi line comment", func(t *testing.T) {
 		tmpl := templateHelper(t)
-		want := `
-  /**
+		want := `/**
```

Comment:
This is not good, it should be indented 1 time (2 spaces).

Reply:
Fixed in https://github.com/dagger/dagger/pull/4367/commits/516b085d5e82abfa2961d86e42104adbf679134a